server/src/main/resources/routes

```diff
@@ -82,6 +82,7 @@ GET           /export/desk-recs/:startLocalDate/:endLocalDate/:terminalName
 GET           /export/desk-recs/snapshot/:localDate/:pointInTime/:terminalName        controllers.application.exports.DesksExportController.exportDesksAndQueuesRecsAtPointInTimeCSV(localDate, pointInTime, terminalName)
 GET           /export/desk-deps/:startLocalDate/:endLocalDate/:terminalName           controllers.application.exports.DesksExportController.exportDesksAndQueuesDepsBetweenTimeStampsCSV(startLocalDate, endLocalDate, terminalName)
 GET           /export/desk-deps/snapshot/:localDate/:pointInTime/:terminalName        controllers.application.exports.DesksExportController.exportDesksAndQueuesDepsAtPointInTimeCSV(localDate, pointInTime, terminalName)
+GET           /export/deployments-combined/:startLocalDate/:endLocalDate              controllers.application.exports.DesksExportController.exportTerminalsDesksAndQueuesRecsForDateRangeCSV(startLocalDate, endLocalDate)
```

Comment:
There's a missing route for handling combined deployments for a point in time (when the time machine is switched on)